### ISSUE TITLE
Unable to access large playlists

### ISSUE THREAD

**Issue by LvL3Bulbasaur** [hrkfdn/ncspot#148]
ncspot seems unable to handle really large playlists, I have a playlist with 1071 songs in and ncspot lists it as "0 tracks" and does nothing when trying to play said playlist.

**Comment by hrkfdn** [comment]
Could you provide a link to such a playlist, so I can reproduce and debug this?

**Comment by LvL3Bulbasaur** [comment]
Certainly here you are:
https://open.spotify.com/playlist/78FJ4NGoaeUVFP5Z294TSm?si=0nwVAlMAT22_LCMNaYzEuw

**Comment by hrkfdn** [comment]
Mhh, odd. If I search for that playlist, it loads fine here. Is there anything suspicious in the debug log?

**Comment by LvL3Bulbasaur** [comment]
Hmm odd indeed, I managed to find a playlist with over 5000 songs in it and it loaded just fine so the issue seems to be playlist specific.
Since I posted this ticket I have tried removing old local tracks that was included in the playlist that I had forgotten about and I have also tried removing tracks witch has been removed from Spotify but the issues seems to persist.

When I search for my playlist posted above it shows it as having zero tracks and the log shows a lot of busy messages from hyper.

I'll attach the log.
[debug.log](https://github.com/hrkfdn/ncspot/files/4370434/debug.log)
